locale/import-symbols.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3.5
+# coding: utf-8
+# NVDA symbols and cldr import
+# Import NVDA symbols dictionaries then emojies from CLDR database
+# Copyright (C) 2001, 2019 Brailcom, o.p.s.
```

Comment:
I  believe you write this file completely? Then put your own copyright, not Brailcom's